EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -4103,20 +4103,20 @@ QUEST_LV_0300_20160811_004102	Say that you should think about it some more	조
 QUEST_LV_0300_20160811_004103	Zanas' Resolve(2)	자나스의 결심(2)				
 QUEST_LV_0300_20160811_004104	Say that he should rethink his options	다시 생각해보라고 한다				
 QUEST_LV_0300_20160811_004105	Using Energy Crystal	원기의 결정 사용 중				
-QUEST_LV_0300_20160811_004106	What needs to be done(1)	지금 해야 할 일은(1)				
+QUEST_LV_0300_20160811_004106	What Must Be Done (1)	지금 해야 할 일은(1)				
 QUEST_LV_0300_20160811_004107	Say that you will not fail	반드시 해내겠다고 한다				
 QUEST_LV_0300_20160811_004108	Say that it is impossible	불가능하다고 한다				
-QUEST_LV_0300_20160811_004109	What needs to be done(2)	지금 해야 할 일은(2)				
+QUEST_LV_0300_20160811_004109	What Must Be Done (2)	지금 해야 할 일은(2)				
 QUEST_LV_0300_20160811_004110	Only activate Kadumel's Magic Circles{nl}You must start over if you activate any of the others	카듀멜의 마법진만 활성화하세요{nl}다른 마법진을 건드리면 처음부터 다시 진행해야 합니다				
 QUEST_LV_0300_20160811_004111	Illusions have been summoned in the Interrogation Room after the Magic Device was activated	마법 장치가 가동되어 심문실에 환영이 생성되었습니다				
-QUEST_LV_0300_20160811_004112	What needs to be done(3)	지금 해야 할 일은(3)				
+QUEST_LV_0300_20160811_004112	What Must Be Done (3)	지금 해야 할 일은(3)				
 QUEST_LV_0300_20160811_004113	Activating the Device	장치를 작동하는 중				
 QUEST_LV_0300_20160811_004114	Opening the Device	장치를 여는 중				
-QUEST_LV_0300_20160811_004115	What needs to be done(4)	지금 해야 할 일은(4)				
+QUEST_LV_0300_20160811_004115	What Must Be Done (4)	지금 해야 할 일은(4)				
 QUEST_LV_0300_20160811_004116	Use the Worn Gravity Stone on monsters	몬스터에게 낡은 그라비티 스톤을 사용하세요				
-QUEST_LV_0300_20160811_004117	What needs to be done(5)	지금 해야 할 일은(5)				
+QUEST_LV_0300_20160811_004117	What Must Be Done (5)	지금 해야 할 일은(5)				
 QUEST_LV_0300_20160811_004118	Use the Worn Gravity Stone to remove the Demon Summoning Crystal	낡은 그라비티 스톤을 사용해서 마족 소환수정을 제거하세요				
-QUEST_LV_0300_20160811_004119	His name is Zanas	그의 이름은 자나스				
+QUEST_LV_0300_20160811_004119	His Name is Zanas	그의 이름은 자나스				
```

Comment:
capital Name?

Reply:
This is fine. It's a quest name.

Reply:
It's a Quest Objective Name.

For reference, please see similar conventions used:

```
QUEST_LV_0200_20150317_001363
QUEST_LV_0100_20150317_002887
```